tests/.pest/snapshots/Countries/BrazilTest/it_can_calculate_brazil_holidays.snap

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "name": "Dia de Ano Novo",
+        "date": "2024-01-01"
+    },
+    {
+        "name": "Carnaval",
```

Comment:
It's noted as optional holiday in Brazil. What should I understand by this? https://publicholidays.com.br/2024-dates/

Reply:
> It's noted as optional holiday in Brazil. What should I understand by this? https://publicholidays.com.br/2024-dates/

It may or may not be a holiday for professional compensation purposes, but it is a holiday.